grammar/gramext_main.ml

```diff
@@ -0,0 +1,290 @@
+(************************************************************************)
+(*  v      *   The Coq Proof Assistant  /  The Coq Development Team     *)
+(* <O___,, *   INRIA - CNRS - LIX - LRI - PPS - Copyright 1999-2017     *)
+(*   \VV/  **************************************************************)
+(*    //   *      This file is distributed under the terms of the       *)
+(*         *       GNU Lesser General Public License Version 2.1        *)
+(************************************************************************)
+
+open Lexing
+open Gramext_ast
+open Format
+
+let fatal msg =
+  let () = Printf.eprintf "Error: %s\n%!" msg in
```

Comment:
Umm, don't open `Format` then use `Printf`. I suggest you use `Format` everywhere.